paging.go

```diff
@@ -20,3 +20,13 @@ type PlaylistTrackPage struct {
 	PagingMeta
 	Items []*PlaylistTrack `json:"items"`
 }
+
+// PagingAlbums represents an PagingObject in the spotify API for albums
+type PagingAlbums struct {
+	Albums AlbumPage `json:"albums"`
+}
+
```

Comment:
A `Paging` object already exists (line 5). You can add the `Albums` field to that struct and simply use the existing `Search()` function instead of creating a new `SearchAlbums()` function.
```suggestion
```

Reply:
So I should just add a field to the Paging object with something like:
```
type Paging struct {
    Tracks TrackPage    `json:"track"``
    Albums AlbumPage `json:"album"`
}

Reply:
Yep, exactly. There's unfortunately no way to have multiple `json:"item"` tags in Golang, so we can't conform to the Spotify documentation exactly.